pkgs/development/tools/misc/blackfire/default.nix

```diff
@@ -1,53 +1,40 @@
-{ stdenv
-, lib
-, fetchurl
-, dpkg
-, autoPatchelfHook
-, writeShellScript
-, curl
-, jq
-, common-updater-scripts
-}:
+{ stdenv, lib, fetchurl, dpkg, writeShellScript, curl, jq, autoPatchelfHook
+, common-updater-scripts }:
 
-stdenv.mkDerivation rec {
+let
   pname = "blackfire";
-  version = "2.5.2";
+  version = "2.8.0";
 
-  src = fetchurl {
-    url = "https://packages.blackfire.io/debian/pool/any/main/b/blackfire/blackfire_${version}_amd64.deb";
-    sha256 = "1RO5yabSNpIz5lWXngMOZ1S2vtnLEkXIj1ZoIinRrQ0=";
+  sources = {
+    "x86_64-linux" = fetchurl {
+      url =
+        "https://packages.blackfire.io/debian/pool/any/main/b/blackfire/blackfire_${version}_amd64.deb";
```

Comment:
Could you bring this on the same line as the attribute name? There is not really much reason to have it split – the line will be long either way, and you are trading three extra characters for one extra line.